.tooling/lua/tests/run.sh

```
@@ -2,7 +2,10 @@
 # This file is used by docker or local to run all tests
 # output is saved to file and error will show in the CLI
 
-cd /QuestieDB
+# Calculate path relative to this script: tests -> lua -> .tooling -> project root
+SCRIPT_DIR=$(dirname "$0")
+cd "$SCRIPT_DIR/../../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)